test/debug_test_application.py

```diff
@@ -13,6 +13,6 @@
 if __name__ == "__main__":
     threadPool = reactor.getThreadPool()
     threadPool.adjustPoolsize(3, 5)
-    options = {'wsgi': application}
+    options = {'wsgi': application, 'max_upload_bytes': 1024}
```

Comment:
This is the default?

Reply:
Default is unlimited to maintain compatibility